Source/WTF/Configurations/modulemap.toml

```diff
@@ -1,46 +1,47 @@
 module-name = 'wtf'
-config-macros = ['UCHAR_TYPE']
-requirements = ['cplusplus11']
-
-# Swift's C++ import struggles with some of the templates found in
-# WTF. To avoid Swift trying to import everything, we divide WTF
-# into lots of submodules. This is why we can't use features such
-# as an umbrella directory.
-one-submodule-per-header = true
-
+discovered-headers-module = 'Cpp'
```

Comment:
OK, so what this essentially means is that if a header _isn't_ assigned to a `[module.<x>]` section, it gets pulled into one overarching submodule named `wtf.Cpp`? Seems reasonable!

I wonder if calling it `default-module` is easier to explain than "discovered headers", but I don't feel strongly about it either way.

Reply:
Exactly. And happy to rename, I'll do that